test/unit/hostgroup_test.rb

```diff
@@ -345,7 +345,7 @@ def setup_user(operation)
 
   test "hostgroup name can't be too big to create lookup value matcher over 255 characters" do
     parent = FactoryGirl.create(:hostgroup)
-    min_lookupvalue_length = "hostgroup=".length + parent.title.length + 1
+    min_lookupvalue_length = 10 + parent.title.length + 1
```

Comment:
It's `Performance/FixedSize`. It's meant to speed up code by using actual values instead of calling a method.